metrotranscom/doorway
# refactor: Align Icon

# Pull Request Template

## Issue Overview

This PR addresses [#36](https://github.com/bloom-housing/Bloom-Doorway/issues/36)


## Description

PR removes Icon / Icons components. And in place of new icons uses font awesome ones.
Those are `Minus` (and Plus to be consistent), and `ExternalLink`.
It was approved by Em.

## How Can This Be Tested/Reviewed?

New `plus` / `minus` icons are on public `/listings` on map zoom controls.
ExternalLink icons are in listing view, below map, next to `view on map`, and on partners site when editing listing in red box, below `To add listings for other Bay Area counties, use one of these portals`.
Make sure that all icons are working as before.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
